src/tracker.md

```diff
@@ -4,11 +4,11 @@ A tracker in this context just means a data type that's able to track changes to
 
 Relm4 does not promote any implementation of a tracker. You're free to use any implementation you like, you can even implement a tracker yourself. In this example however, we'll use the tracker crate that provides a simple macro that implements a tracker for us automatically.
 
-In this chapter we are going to implement a small program using this technique which displays two randomly picked icons on two buttons:
+In this chapter we are going to implement a small program using this technique which displays two randomly picked icons which are controlled by two buttons:
```

Comment:
Good idea. Yet I realized that the original sentence was a bit confusing so I suggest to move some parts of the sentence to make it more clear:

Using this technique, we will implement a small program which displays two randomly picked icons that are controlled by two buttons.